modules/serverless-broker-backing-channel-default.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-backing-channel-default_{context}"]
+= Configuring the default broker backing channel
+
+If you are using a channel-based broker, you can set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.
```

Comment:
```suggestion
If you use a channel-based broker, you can set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.
```

> set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.

Why/when should I do this? Consider adding explanation or something for information scent if the audience won't be absolutely sure.

Reply:
The differences between the channel implementations are explained here: https://docs.openshift.com/container-platform/4.11/serverless/discover/serverless-channels.html#serverless-channels-implementations

Tbh there's no reason why we'd recommend someone to use this, since we'd prefer them to move to the Kafka broker entirely (we've been moving to recommending that in our docs), but it was requested from eng that we document it anyway, since it's one of the main configuration options for this type of broker. If a user is using a channel-based broker over the Kafka broker, it can be assumed that they would understand why they would want to configure this, because the use case would be using channels that aren't the Kafka channel.